### PR TITLE
Fixed documentation for dialog

### PR DESCRIPTION
**Pull request by FabienLelaquais**
Addresses #1115 and #1148.

Renamed all references to code examples

### CODE REVIEW

File: docs/refmans/gui/viselements/generic/date_range.md_template [hunk 1]
```diff
@@ -112,3 +108,8 @@ Here is how the date range selector looks like:
     <img src="../date_range-labels-l.png" class="visible-light"/>
     <figcaption>Labels on the date fields</figcaption>
 </figure>
+
+# Styling
+
+All the date range controls are generated with the "taipy-date_range" CSS class. You can use this
```
Comment (FredLL-Avaiga) [inline docs/refmans/gui/viselements/generic/date_range.md_template]: Typo taipy-date_range

**Review by FredLL-Avaiga — APPROVED** [review]
I suppose I'm wrong about taipy-date_range but it looks really weird... I suppose for 5.0 :-)
Well done
Quite a job

**Review by namnguyen20999 — APPROVED** [review]
lgtm

File: docs/refmans/gui/viselements/generic/date_range.md_template (hunk 1 above)
Comment (FabienLelaquais) [reply docs/refmans/gui/viselements/generic/date_range.md_template]: Sorry I don't see the typo.

Comment (FabienLelaquais) [reply docs/refmans/gui/viselements/generic/date_range.md_template]: Oh.
```suggestion
All the date range controls are generated with the "taipy-date-range" CSS class. You can use this
```

Got it

File: docs/refmans/gui/viselements/generic/file_download.md_template [hunk 2]
```diff
@@ -195,3 +182,16 @@ sets the *on_action* parameter that indicates that, when the file transfer is pe
 function *clean_up()* should be called.<br/>
 In *clean_up()*, we simply delete the temporary file, retrieving its path name from the provided
 *state*.
+
+# Styling
+
+All the file download controls are generated with the "taipy-file_download" CSS class. You can use this class
```
Comment (FredLL-Avaiga) [inline docs/refmans/gui/viselements/generic/file_download.md_template]: _ => -

File: docs/refmans/gui/viselements/generic/file_selector.md_template [hunk 3]
```diff
@@ -50,3 +37,16 @@ True.
 !!! taipy-element
     default={content}
     multiple:b=True
+
+# Styling
+
+All the file selector controls are generated with the "taipy-file_selector" CSS class. You can use this class
```
Comment (FredLL-Avaiga) [inline docs/refmans/gui/viselements/generic/file_selector.md_template]: idem

**Review by FredLL-Avaiga — CHANGES_REQUESTED** [review]
_ => - in class names

File: docs/refmans/gui/viselements/generic/file_download.md_template (hunk 2 above)
Comment (FabienLelaquais) [reply docs/refmans/gui/viselements/generic/file_download.md_template]: Ok

File: docs/refmans/gui/viselements/generic/file_selector.md_template (hunk 3 above)
Comment (FabienLelaquais) [reply docs/refmans/gui/viselements/generic/file_selector.md_template]: Ok

**Review by FredLL-Avaiga — APPROVED** [review]
👍

**Pull request merged**